addons/cartel/CfgVehicles_Soldados.hpp

```diff
@@ -0,0 +1,131 @@
+    // Units - Cartel Soldados
+
+    class TACU_Cartel_U_Soldier_Base: TACU_Main_U_OPFOR_Soldier_Base {
```

Comment:
`TACU_Main_U_OPFOR_Soldier_Base` already acts as a base, so `TACU_Cartel_U_Soldier_Base` can be replaced with `TACU_Cartel_U_Soldado_Rifleman_01` (and following Cartel Soldados can be based on rifleman 01)